th/lessons/advanced/erlang.md

```diff
@@ -0,0 +1,138 @@
+---
+version: 1.0.0
+title: การใช้งานร่วมกับ Erlang
+redirect_from:
+  - /lessons/advanced/erlang/
+---
+
+หนึ่งในประโยชน์ของการสร้างบน Erlang VM (BEAM) คือมี library ให้เลือกใช้มากมายก่ายกอง ความสามารถในการทำงานร่วมกับ ทำให้เราเราสามารถใช้ library และ Erlang standard lib ได้ใน code Elixir ในบทนี้เราจะมาดูวิธีการใช้งาน funtion ใน standard lib ไปพร้อม ๆ กับ แพ็คเกจเสริมของ Erlang
+
+{% include toc.html %}
+
+## Standard Library
+
+standard library เสริมของ ของ Erlang สามารถใช้งานได้ทุกที่ใน code Elixir ของ application เรา. Module Erlang จะอยู่ในรูปของ atom แบบ lowercase เช่น `:os`, และ `:timer`
```

Comment:
- Remove the full stop after "เรา". We don't have it in Thai. :P
- Module -> โมดูล
- เช่น `:os`, และ `:timer` -> เช่น `:os` และ `:timer`